app/helpers/application_helper.rb

```diff
@@ -80,6 +80,7 @@ def note_box_color(type)
   end
 
   def status_badge(status, style, pulse: false)
+    style = STATUS_COLOR_PALETTE[style] if style.is_a?(Symbol)
```

Comment:
why?

Reply:
see [this](https://github.com/tramlinehq/tramline/pull/368/files#diff-565bb985fdb2bc0e19dc492717b734d6ac62b932c367d023d6cd0d8ba52a9a00R98)